steampipecloud/table_steampipecloud_organization.go

```diff
@@ -110,7 +110,7 @@ func listOrganizations(ctx context.Context, d *plugin.QueryData, h *plugin.Hydra
 	// execute list call
 	pagesLeft := true
 
-	var resp openapi.ListActorOrgsResponse
+	var resp openapi.ListUserOrgsResponse
```

Comment:
@siddharthaturbot Is this change related to the table that this PR is adding? If not, can you please describe why we're making this change?